<div>
  <h3 class="md-headline">About the Genomic Observatories Metadatabase</h3>
    <div>
      <p>
        The Genomic Observatories Meta-Database (GeOMe) is a web-based database which captures metadata on biological samples, used
        for biodiversity inventories, population studies, and environmental metagenomics. GeOMe assigns persistent identifiers
        for all samples and sampling events and specifies the set of metadata attributes which satisfy the requirements of
        the
        <a href="https://gigascience.biomedcentral.com/articles/10.1186/2047-217X-3-2">genomic observatories model</a>, 
	including capturing the who, what, where, and when associated with all samples.
      </p>	
      <p>
       <b>Table of Contents:</b>
       <ul>
       <li><a href="#workflowDiagram">GeOMe Workflow Diagram</a></li>
       <li><a href="#downloadVideo">Search and Download Data Video</a></li>
       <li><a href="#uploadVideo">Upload Metadata Video</a></li>
       <li><a href="#templates">Template generation: what you need to do before you upload data</a></li>
       <li><a href="#upload">Users can upload several types of data</a></li>
       <li><a href="#rpackage">Query GeOMe using "R"</a></li>
       <li><a href="#whyfastq">Why FASTQ submissions for next generation sequence data?</a></li>
       <li><a href="#limsPlugin">About the Biocode LIMS plugin</a></li>
       <li><a href="#dataUsage">Data usage policy</a><//li>
       <li><a href="#funding">Funding</a></li>	
       </ul>
      </p>
    </div>
    <md-divider class="margin-bottom"></md-divider>
</div>

<div>
  <h3 id="workflowDiagram" class="md-headline">GeOMe Workflow Diagram</h3>
    The following diagram shows how data is managed in the GeOMe environment.  Boxes in solid blue indicate GeOMe functions that the user interacts with, such as generating templates and loading data.  The white boxes with a green border are functions that happen outside of the GeOMe environment, such as data entry into a spreadsheet.
    <div flex>
      <img id='workflowImage' class="img-responsive" style="max-height: 350px;" src='docs/geomeWorkflow.png'>
    </div>
</div>

<div>
  <h3 id="downloadVideo" class="md-headline">Search and Download Data Video</h3>
  The following video illustrates how to search and download data from Version 1 of the GeOMe interface.  The screens may appear different in current versions of the software but the instructions will still be helpful.
  <iframe width="560" height="315" src="https://www.youtube.com/embed/rdoP7xqDyhQ" frameborder="0" allowfullscreen></iframe>
</div>

<div>
  <h3 id="uploadVideo" class="md-headline">Upload Metadata Video</h3>
  The following video illustrates how to upload metadata to Version 1 of the GeOMe interface.  The screens may appear different in current versions of the software but the instructions will still be helpful.
  <iframe width="560" height="315" src="https://www.youtube.com/embed/GgPsFnoinZc" frameborder="0" allowfullscreen></iframe>
</div>

<div>
  <h3 id="templates" class="md-headline">Template generation: what you need to do before you upload data</h3>
  <ul>
    <li>
      Generate a template (see Workbench -> "Generate Template") to create a spreadsheet template for your metadata
      <ul>
        <li>
          This option allows the user to create an excel file to store the metadata associated with each of their samples.
        </li>
      </ul>
    </li>
  </ul>
</div>

<div>
  <h3 id="upload" class="md-headline">Users can upload several types of data</h3>
  <p>To upload data users much have an account. If you don't have one please send an email with you name, title and affiliation
    to
    <a href="mailto:geome.database@gmail.com">geome.database@gmail.com</a>
  </p>
  The following types of data are recognized in the GeOMe system:
  <ol>
    <li>Collecting Event metadata
    <li>Sample metadata
    <li>Tissue metadata
    <li>Photos of collecting event metadata
    <li>Photos of sample metadata
    <li>Published
      <u>Sanger sequence data</u> and the associated metadata (fasta format)</li>
    <li>
      Metadata associated with
      <u>next-generation sequence data</u> (fastq format)
      <ul>
        <li>In the process of uploading the metadata for your next-generation sequence data we create two files:
        </li>
        <ol>
          <li>the SRA metadata file</li>
          <li>the BioSample datasheet</li>
        </ol>

      </ul>
    </li>
  </ol>
  <p>
    <strong>This will make uploading your data to Genbank's Sequence Read Archive (SRA) database faster and easier
    </strong>.</p>
  <p>
    <small>* Note we are not currently accepting microsatellite datasets</small>
  </p>
</div>


<div>
  <h3 id="rpackage" class="md-headline">Query GeOMe using "R"</h3>
    <p>You can query datasets that have been loaded to the GeOMe database using our
       <a href="https://github.com/DIPnet/fimsR-access">R package</a>.  Please note, the R package was released for version 1 of 
       the GeOMe site and has not yet been fully tested with the current release.  If you encounter issues, please contact the
       development team.
    </p>
</div>

<div>
  <h3 id="whyfastq" class="md-headline">Why FASTQ submissions for next generation sequence data?</h3>
  <p>GeOMe accepts FASTQ submissions because we want to archive unmanipulated sequence data that are free of filtering biases.
    By doing this we are reducing ascertainment bias (a site that is not a SNP in your dataset may be a SNP when combined
    with data from a different population). We are also avoiding the subjective choices that go into calling SNPs, such as
    thresholds for trimming, filtering, coverage and likelihood. Our objective is for future users of your data to be able
    to make these choices in the context of their own question and dataset.
  </p>
</div>

<div>
  <h3 id="limsPlugin" class="md-headline">The Biocode LIMS Plugin</h3>
      <p style="margin-top:20px;">
       Data uploaded to GeOMe can be manipulated in a laboratory information management system (LIMS) using 
       a specially built <a href="http://software.mooreabiocode.org/">LIMS plugin</a> that operates within the Geneious environment.
       The purpose of the LIMS tool is to help manage lab and sequence analysis workflows. Note the name of the "LIMS plugin" for GeOMe
       is referred to as the "Biocode LIMS Plugin" since it was originally developed for the Moorea Bioocode Project.  It has
       since been adapted for use in the GeOMe environment.
      </p>
</div>

<div>
  <h3 id="dataUsage" class="md-headline">Data usage policy</h3>
      <p style="margin-top:20px;">GeOMe data is subject to the data usage policy developed by the Diversity of the Indopacific Project (the 
      first GeOMe project).  The data usage policy can be found 
        <a target="_blank" href="http://diversityindopacific.net/data-usage-agreement/">here</a>
      </p>
</div>

<div>
  <h3 id="funding" class="md-headline">Funding</h3>
  <p>
    Development of GeOMe has been supported by the
      <a href='https://www.moore.org/article-detail?newsUrlName=ambitious-project-to-create-genetic-catalog-of-a-model-ecosystem-gets-$5-2-million-from-the-gordon-and-betty-moore-foundation'>Gordon and Betty Moore Foundation</a>, the
      <a href='https://bnhm.berkeley.edu/'>Berkeley Natural History Museums</a>, the National Science Foundation (
      <a href='https://www.nsf.gov/awardsearch/showAward?AWD_ID=0956426&HistoricalAwards=false'>DEB-0956426</a> and
      <a href='https://nsf.gov/awardsearch/showAward?AWD_ID=1457848'>DEB-1457848</a>), and contracts from the
      <a href='https://naturalhistory.si.edu/'>Smithsonian National Museum of Natural History</a>.
  </p>
</div>
