<div>
  <h2>About the Genomic Observatories Metadatabase</h2>
  <p>
    The Genomic Observatories Meta-Database (GeOMe) is a web-based database which captures metadata on biological
    samples, used
    for biodiversity inventories, population studies, and environmental metagenomics. GeOMe satisfies the requirements
    of
    the
    <a href="https://gigascience.biomedcentral.com/articles/10.1186/2047-217X-3-2">genomic observatories model</a>,
    including capturing the who, what, where, and when associated with all samples. GeOMe
    draws on both the
    <a href="http://rs.tdwg.org/dwc/">Darwin Core</a> and the
    <a href="http://gensc.org/mixs/">Minimum Information about any type of Sequence (MIxS)</a> standards.
  </p>
</div>
<br>
<uib-accordion>
  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="Getting Started">
    <p>
    Welcome to GeOMe! There are several routes you can take to get started with GeOMe.  The route you take depends on whether you
    are interested in only adopting an available template, generated from a network approved configuration, or if you want to submit 
    data to the GeOMe repository.  A network approved configuration is a configuration template that projects can adopt during creation.  
    Currently, there are two network approved configurations: one for <b>Biocode</b> projects and another for <b>GeOMe publications</b> (which follow the
    model developed by the Diversity of the IndoPacific Network for publishing genomic data.
    Templates are a collection of data attributes, rules, and lists that are represented through the 
    interface as JSON files, with the option of generating Excel spreadsheets.  JSON files are useful for developers in automating the discovery
    of metadata attributes belonging to a template while Excel spreadsheets are useful for guiding the data attributes for a particular
    task.
    </p>

    <p>
    If you choose to submit data to the GeOMe repository, you can choose to submit to an 
    existing project, create a new project that is synchronized to a network approved configuration, or create a new project either from 
    scratch or cloned but un-synchronized with a network approved configuration.   When a project is synchronized with a network 
    approved configuration, all changes to the master configuration (e.g. Biocode), will apply to all projects that belong to the 
    master configuration.  Projcts that are cloned but un-synchronized with the network approved configuration will not receive any 
    configuration updates once they are created.
    </p>

    <p>
    The following lists the available routes you can choose to get started:
    </p>

    <p>
      <strong><u>Route 1</u>: I want to query public data or generate a template</strong>
      <br>Most GeOMe projects are marked as "public", which means that anyone can query public data or generate that
      project's templates.  Read the <a href='docs/helpDocumentation.pdf' target=_blank>GeOMe Help Document</a> to get started 
      using the web interface for generating templates and querying data.</br>
    </p>

    <p>
      <strong><u>Route 2</u>: I want to contribute data to an existing project</strong>
      <br>See the <a ng-click="$ctrl.openProjects()">GeOMe projects page</a> to read about current GeOMe projects. 
      Contact the project administrator of the project you would like to join and the will either create a new account 
      for you or associate your existing account to their project.  Read the 
      <a href='docs/helpDocumentation.pdf' target=_blank>GeOMe Help Document</a> to get started using the web interface.</br>
    </p>

    <p>
      <strong><u>Route 3</u>: I want to create a new project, synchronized with a network approved configuration</strong>
      <br>If you choose to create a project that is synchronized with a network approved configuration, you will receive all
      updates and additions of data attributes, rules, and lookkup lists of the network approved configuration. 
      Creating any new project currently requires sending an email request to <a href="mailto:geome.help@gmail.com">geome.help@gmail.com</a> 
      with a brief statement of what your project involves.</br>
    </p>

    <p>
      <strong><u>Route 4</u>: I want to create a new project, un-synchronized with a network approved configuration</strong>
      <br>If you choose to create a project that is un-synchronized with a network approved configuration, you will either start
      from scratch or you can choose to clone a network approved configuration.  If your project is un-syncrhonized, then you are
      free to make your own changes and additions to your project configuration and you will not receive any changes made by the
      master network approved configuration.  Creating a new project currently requires sending an email request to 
      <a href="mailto:geome.help@gmail.com">geome.help@gmail.com</a> with a brief statement of what your project involves.</br>
    </p>

    <p>
      The <a href='docs/helpDocumentation.pdf' target=_blank>GeOMe Help Document</a> contains detailed instructions on using the web interface.
    </p>
  </div>

  <div uib-accordion-group id="projects" class="panel-default md-whiteframe-2dp" heading="Projects and Configurations" is-open="$ctrl.projects">
    <p>
    The following network approved configurations and projects are active in the GeOMe database.  It is important for new users to be familiar with existing
    projects in GeOMe before getting started.  Please go through this list to see if you can join an existing project before 
    creating a new project. When joining a project you agree to work together with the project administrator and project users.
    The following list of projects describes the project, indicates whether it is public or not and gives a contact for getting 
    started.  
    </p>

    <project-info ng-if="$ctrl.projects"></project-info>

  </div>

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="Using the Web Interface" id="userHelp"
    is-open="$ctrl.userHelp">
    <p>
      GeOMe's web interface lets users create spreadsheet templates, customized for individual projects, which ensure
      rapid feedback on data quality during the data loading process.
      The web interface lets users upload data on collecting events, specimens, tissues,
      photos of specimens and events, FASTA files, and FASTQ metadata. Users can also choose to create their own GeOMe
      projects and customize their own attributes, lookup lists, rules, and project members.
    </p>

    <p>
      Step by step instructions on using the GeOMe web interface are available by reading our
      <a href='docs/helpDocumentation.pdf ' target='_blank '>GeOMe Help Document (PDF format)</a>.</p>
  </div>

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="Workflow Diagram">
    <p>The following diagram shows how data is managed in the GeOMe environment. Boxes in solid blue indicate GeOMe
      functions
      that the user interacts with, such as generating templates and loading data. The white boxes with a green border
      are
      functions that happen outside of the GeOMe environment, such as data entry into a spreadsheet. </p>
    <div flex>
      <img id='workflowImage ' class="img-responsive" style="max-height: 350px;" src='docs/geomeWorkflow.png '>
    </div>
  </div>


  <!-- <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading='Query GeOMe using "R"'>
    <p>You can query datasets that have been loaded to the GeOMe database using our
      <a href="https://github.com/DIPnet/fimsR-access">R package</a>. Please note, the R package was released for version 1 of the GeOMe site and has not yet been fully tested
      with the current release. If you encounter issues, please contact the development team.
    </p>
  </div> -->

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="The Biocode LIMS Plugin">
    <p>
      Data uploaded to GeOMe can be manipulated in a laboratory information management system (LIMS) using a specially
      built
      <a href="http://software.mooreabiocode.org/">LIMS plugin</a> that operates within the Geneious environment. The
      purpose of the LIMS tool is to help manage lab and
      sequence analysis workflows.Use the
      <a href="https://github.com/biocodellc/biocode-lims/wiki">LIMS plugin wiki</a> to learn about how to use the LIMS
      and download the latest version from the
      <a href="https://github.com/biocodellc/biocode-lims/releases">Biocode LIMS Release page</a>. After you have
      installed the plugin and learned how to open a connection from the wiki
      docs, specify the following to make your connection:
      <ul>
        <li>
          <strong>Field Database Connection:</strong> GeOMe FIMS</li>
        <li>
          <strong>Host:</strong> https://api.develop.geome-db.org (currently only the development version of the LIMS
          is specified).</li>
        <li>
          <strong>Username/password:</strong> You will need to obtain a username and password in order to make any
          connection to
          the LIMS system.</li>
      </ul>
    </p>
  </div>

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="Instructional Videos">
    <p>
      While the content of these videos reference a prior version of the web interface, the functionality is still
      applicable to the current interface. The videos cover generating templates, loading data, and associating FASTQ
      sequence data and loading to the Sequence Read Archive.
    </p>

    <iframe width="560" height="315" src="https://www.youtube.com/embed/rdoP7xqDyhQ" frameborder="0" allowfullscreen></iframe>
    <iframe width="560" height="315" src="https://www.youtube.com/embed/GgPsFnoinZc" frameborder="0" allowfullscreen></iframe>
  </div>

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="History">
    <p>
    GeOMe has its roots in the Moorea Biocode project database, developed from 2006 to 2011 to support data collection for the 
    Moore Foundation funded Moorea Biocode Project: an all taxa biotic inventory of a single tropical island involving 6 teams, 50 researchers,
    and thousands of collecting events.  The Moorea Biocode field information management system, also known as "FIMS1", was developed
    to ingest spreadsheet data from researchers working on the project and employed data validation on data ingest.  The tools were
    written on server side with Perl with a client application in Java (bioValidator).   FIMS1 has been running from 2006 through 2018 and 
    developed tools such as the Plate Matcher (to easily map tissues to 96 well plates), the bioValidator (for loading and validating spreadsheets), 
    and a web interface for managing data.
    </p>
    <p>
    From 2012 to 2015, the National Science Foundation funded the BiSciCol (Biological Sciences Collections) project.
    While BiSciCol focused on identifiers, ontologies, and semantic technologies for linking biodiversity data, the primary use case
    was to work on a solution for linking events, samples, and tissues across many different systems using linked data technology.  The project
    had mixed results but did result in several achievements: a clearer understanding of the role of persistent identifiers, building 
    a darwin core triplifier, the development
    of the Biological Collections Ontology (BCO), and FIMS2 (running at http://biscicol.org/).  FIMS2 collaborated with the California
    Digital Library system to assign ARK identifiers (through their EZID system) for all samples, events and tissues.   In addition,
    all data inserted into FIMS2 was backed by a Fuseki triplestore with all metadata directly assigned data attributes identified by 
    Darwin Core and the MIxS Standards.  
    </p>
    <p>
    From 2014-2017, FIMS2 also began working with the Diversity of the IndoPacific Network (DIPNet) to develop a coherent metadata repository for
    assembling metadata from across the IndoPacific region.  DIPNet added the ability to upload FASTA and FASTQ metadata to the FIMS2 site and also 
    developed the name "GeOMe", which eventually became the brand name for FIMS3.  As of 2018, FIMS2 is still running at http://biscicol.org/ but is being transitioned to FIMS3.
    </p>
    <p>
    Beginning in 2016, John Deck and RJ Ewing started work on a redesigned FIMS system, utilizing a Postgres backend to control project management features and used JSON metadata 
    objects to store data and configurations.   This FIMS system (FIMS3) assumed the name GeOMe.   It incorporates design features from FIMS1 (including photo uploading,
    specimen, tissue, and event pages, plate-matching tools) along with features of FIMS2 (persistent id generation, flexible project configuration with attributes stored
    in a non-relational system guided by a configuration file).  In addition, FIMS3 brings in the concept of a network which governs multiple configuraiton templates and the 
    ability to easily create projects.   GeOMe will be replacing FIMS1 and FIMS2 instances before the end of 2018.
    </p>
  </div>

  <div uib-accordion-group class="panel-default md-whiteframe-2dp" heading="Data usage policy" id="dataPolicy" is-open="$ctrl.dataPolicy">
    The Genomic Observatories Metadatabase has the following objectives:

    <ul>
      <li>To advance genetic diversity research worldwide</li>
      <li>To aggregate sample and genetic data in raw formats in a searchable database such that original datasets can
        be utilized for further investigation.</li>
      <li>To promote and advocate open and collaborative science as a best practice for conducting biodiversity
        research</li>
      <li>To promote and provide capacity-building within developing countries for monitoring, study and protection of
        their biodiversity resources.</li>
    </ul>

    <p>
      Recalling that access to and utilization of genetic resources and data taken should be consistent with the
      provisions of the Convention on
      Biological Diversity (CBD) taking into account their specifications by the Bonn Guidelines on Access to Genetic
      Resources and the Fair and
      Equitable Sharing of Benefits arising from their Utilization, and, where appropriate, the Nagoya Protocol on
      Access to Genetic Resources and the
      Fair and Equitable Sharing of Benefits arising from their Utilization (NP),
    </p>

    <p>
      Recalling that according to these provisions non-monetary and/or monetary benefits from the utilization of the
      genetic resources shall be
      shared with the Country of Origin if the same so requires and as it is set out in mutually agreed terms,
    </p>

    <p>
      Acknowledging that research and development on genetic resources can be for the public domain (non-commercial) or
      for commercial purposes and,
    </p>

    <p>
      Recalling that according to these provisions, non-commercial research purposes may contribute to the conservation
      and sustainable use of biodiversity
    </p>

    <p>
      By uploading data to or downloading data from the DIPnet database, DIPnet contributors and Recipients of DIPnet
      data agree as follows:
    </p>

    <p><b>ARTICLE 1 – UPLOADING OF LEGAL AND ACCURATE GENETIC DATA AND ASSOCIATED METADATA</b></p>

    <p>
      By uploading Genetic Data and associated metadata to GeOME database, Contributors certify the following:
      <p><b>1.1</b> The Genetic Resources from which Genetic Data and Associated Metadata were derived were collected under 
      appropriate and legal access permits or their equivalent by each Country of Origin and that the data contributor can 
      provide their copies upon request, and </p>
      <p><b>1.2</b> All co-authors on the data’s primary publication have consented to share the data in the GeOMe database.</p>
    </p>

    <p><b>ARTICLE 2 – USAGE OF DOWNLOADED GENETIC DATA AND ASSOCIATED METADATA</b></p>

    <p><b>2.1</b> The Recipient shall be entitled to the Use of the Genetic Data and Associated Metadata for the Public
      Domain.</p>

    <p><b>2.2</b> Should the Recipient intend to utilize the Genetic Data and Associated Metadata for Commercial
      Purposes they will seek consent of the Country of Origin, subject to the mutually agreed terms between the DIPnet
      contributor and Country of Origin.</p>

    <p><b>2.3</b> When used for Commercial Purposes or in the Public Domain, data should be recognized as follows:
      “Data were made available through GeOMe (https://geome-db.org/)”.</p>

    <p><b>2.4</b> Every attempt should be made to cite the original studies having contributed data to the new
      analyses. Attribution would include any publications first describing these data as well as reference to
      electronic depositions (such as DOI associated with dataDryad). This information may often be found in the
      “associatedReferences” field of the database.</p>

    <p><b>2.5</b> The Recipient may transfer downloaded Genetic Data and Associated Metadata to Subsequent Recipients
      provided that Subsequent Recipients agree in writing to use the data under the terms of this agreement.</p>

    <p><b>2.6</b> GeOMe does not undertake to monitor the rights of any Country of Origin for their Genetic Resources
      downstream from our database portal.</p>

  </div>
</uib-accordion>

<div>
  <p>
    <a href="https://doi.org/10.1371/journal.pbio.2002925">
      <i>"The Genomic Observatories Metadatabase (GeOMe): A new repository for field and sampling event metadata
        associated
        with genetic samples"</a>, John Deck , Michelle R. Gaither, Rodney Ewing, Christopher E. Bird, Neil Davies,
    Christopher
    Meyer, Cynthia Riginos, Robert J. Toonen, Eric D. Crandall Published: August 3, 2017
    <br>
    <a href="https://doi.org/10.1371/journal.pbio.2002925">https://doi.org/10.1371/journal.pbio.2002925</a>
    </i>
  </p>

  <p>
    Development of GeOMe has been supported by the
    <a href='https://www.moore.org/article-detail?newsUrlName=ambitious-project-to-create-genetic-catalog-of-a-model-ecosystem-gets-$5-2-million-from-the-gordon-and-betty-moore-foundation '>Gordon
      and Betty Moore Foundation</a>, 
      the National Science Foundation ( <a href='https://www.nsf.gov/awardsearch/showAward?AWD_ID=0956426&HistoricalAwards=false '>DEB-0956426</a> and <a href='https://nsf.gov/awardsearch/showAward?AWD_ID=1457848 '>DEB-1457848</a>), 
      the <a href='https://bnhm.berkeley.edu/ '>Berkeley Natural History Museums</a>, 
      and contracts from the <a href='https://naturalhistory.si.edu/ '>Smithsonian National Museum of Natural History</a>.
  </p>
</div>
